docs/user/setup.rst

```diff
@@ -48,111 +57,175 @@ You can read the installation instructions by clicking one of these logos below:
   :width: 240px
-  :target: #via-docker
+  :target: #via-singularity
 
 |
 
-Docker is a virtualization software packaging applications and its dependencies in a virtual container that can run on any Linux server.
-In fact, it is available for a wide variety of operating systems, e.g. Linux, Mac and Windows. For more information on this technology, take a look at `this overview <https://www.docker.com/resources/what-container>`__.
-
+Singularity is an application container for **Linux** systems.
+For more information read the full documentation of the Singularity
+`here <https://sylabs.io/docs/>`__.
 
-1. Pull the NEST Desktop image from Docker Hub:
+1. Clone a working copy from the repository and go to the folder:
 
 .. code-block:: bash
 
-  docker pull nestdesktop/app
+  git clone https://github.com/nest-desktop/nest-desktop-singularity
+  cd nest-desktop-singularity
 
-2. Start the Docker container:
+2. Register bash command for NEST Desktop Singularity:
 
 .. code-block:: bash
 
-  docker run -rm -it -p 8000:8000 --name nest-desktop nestdesktop/app
+  export PATH=$PATH:$PWD/bin/
 
-NEST Desktop is now serving at ``http://localhost:8000``.
+3. Build Singularity images (it will ask for sudo password):
 
-.. note::
+.. code-block:: bash
+
+  nest-desktop-singularity build
 
-  It only starts NEST Desktop without NEST Server.
-  If you want to both together (front end and back end), you can use Docker Compose, which is documented :doc:`here </deployer/deploy-docker>`.
+4. Start Singularity instances of NEST Desktop and NEST Simulator:
 
+.. code-block:: bash
 
-.. rubric:: Arguments
+  nest-desktop-singularity start
 
-You can find the help texts of docker arguments by :code:`docker run --help`.
+Now NEST Desktop is started.
+You can use NEST Desktop in the web browser at http://localhost:8000.
 
-+----+-------------------------------------------+
-| -p | Publish a container's port(s) to the host |
-+----+-------------------------------------------+
-| -i | Keep STDIN open even if not attached      |
-+----+-------------------------------------------+
-| -t | Allocate a pseudo-TTY                     |
-+----+-------------------------------------------+
+**Installation is now complete!**
+:doc:`Now we can start constructing networks for the simulation! <usage>`.
 
+For more information read the full documentation of the `NEST Desktop Singularity <https://github.com/nest-desktop/nest-desktop-singularity>`__.
 
 
-Via Singularity
----------------
 
-.. image:: ../_static/img/logo/singularity-logo.svg
+Via Python Package for |linux| |windows| |apple|
+------------------------------------------------
+
+.. image:: ../_static/img/logo/pypi-logo.svg
   :width: 240px
-  :target: #via-singularity
+  :target: #via-python-package
 
 |
 
-Singularity is an application container for Linux systems.
-For more information read the full documentation
-`here <https://sylabs.io/docs/>`__.
+1. (For advanced user) Install NEST Simulator:
```

Comment:
I'd like to keep your mention that the installation via pip is planned: A persons setting this up will likely find this and is less likely to open a duplicate issue.
```suggestion
NEST Simulator cannot be installed via pip (`maybe soon <https://github.com/nest/nest-simulator/pull/2073>`__).
Therefore, we need to install it in another way.
1. (For advanced users) Install NEST Simulator:
```